taskt/UI/CustomControls/CustomUIControls/UITabControl.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System.Windows.Forms;
+
+namespace taskt.UI.CustomControls.CustomUIControls
+{
+    public class UITabControl : TabControl
+    {
+        //protected override void OnPaint(PaintEventArgs e)
+        //{
+        //    DrawControl(e.Graphics);
+        //}
+
+        //internal void DrawControl(Graphics g)
+        //{
+        //    if (!Visible)
+        //        return;
+
+        //    Brush br = new SolidBrush(Color.Black);
+        //    Brush brTab = new SolidBrush(Color.Orange);
+
+        //    Rectangle TabControlArea = ClientRectangle;
+        //    Rectangle TabArea = DisplayRectangle;
```

Comment:
Whole class body is commented, is it intentional?

Reply:
That was already commented out in the original version. I expect he was intending to make customization to the standard tab control but put it on hold

Reply:
I actually might use some of this code when I finish making the close buttons for tabs so I don't want to delete it yet.

Reply:
Got it.